gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -114,10 +129,13 @@ public void write (FileHandle file) {
 					write.write(buffer, 0, bytesToWrite);
 					writtenBytes += bytesToWrite;
 				}
+
+				write.close();
+
+				gzip.close();
+
 			} catch (Exception e) {
-				throw new GdxRuntimeException("Couldn't write PKM file to '" + file + "'", e);
-			} finally {
-				StreamUtils.closeQuietly(write);

```

Comment:
Why did you remove this?
Now an exception will be thrown when closing the file failed.
Previously nothing would have happened. I don't see any reason to change this behavior.